conda/history.py

```diff
@@ -31,26 +31,65 @@ def is_diff(content):
     return any(s.startswith(('-', '+')) for s in content)
 
 
+def parse_content_line(s):
+    """
+    If a line representing a package addition/removal has more than one word, it is assumed
+    that words 2, 3, and 4 represent name, version, and build, respectively. Otherwise the
+    parser will split apart the dist name according to current convention.
+    """
```

Comment:
Doctests are being included in coverage now.  Excellent place to use them.  In the docstring add an `Examples:` section.

https://github.com/conda/conda/blob/4.3.x/conda/common/url.py#L111-L119

Well, maybe they're not yet being run.  Because then there'd obviously be coverage there.  I'll fix that though.